src/Microsoft.AspNet.Authentication.Cookies/CookieServiceCollectionExtensions.cs

```diff
@@ -8,10 +8,16 @@
 namespace Microsoft.Extensions.DependencyInjection
 {
     /// <summary>
-    /// Extension methods provided by the cookies authentication middleware
+    /// Extension methods for setting up cookie authentication services in an <see cref="IServiceCollection" />.
     /// </summary>
     public static class CookieServiceCollectionExtensions
     {
+        /// <summary>
+        /// Adds cookie authentication services to the specified <see cref="IServiceCollection" />. 
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
+        /// <param name="configure">An action delegate to configure the provided <see cref="CookieAuthenticationOptions"/>.</param>
+        /// <returns>A reference to this instance after the operation has completed.</returns>
         public static IServiceCollection AddCookieAuthentication(this IServiceCollection services, Action<CookieAuthenticationOptions> configure)
```

Comment:
Yeah I will go through and nuke these, they are worse than obsolete, they pretend to do stuff :)